jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -93,21 +93,19 @@ private Tracer(
     this.zipkinSharedRpcSpan = zipkinSharedRpcSpan;
     this.activeSpanSource = activeSpanSource;
 
-    int ip;
-    try {
-      ip = Utils.ipToInt(Inet4Address.getLocalHost().getHostAddress());
-    } catch (UnknownHostException e) {
-      ip = 0;
-    }
-    this.ip = ip;
-
     this.version = loadVersion();
 
     tags.put("jaeger.version", this.version);
     String hostname = getHostName();
     if (hostname != null) {
       tags.put("jaeger.hostname", hostname);
     }
+    int ipv4 = 0;
+    try {
+      tags.put("ip", InetAddress.getLocalHost().getHostAddress());
```

Comment:
Also, I'd recommend putting `InetAddress.getLocalHost().getHostAddress()` into a variable because the host address might change due to DHCP.